code/game/machinery/vending.dm

```diff
@@ -269,7 +275,8 @@
 /obj/machinery/vending/emag_act(user as mob)
 	if(!emagged)
 		emagged  = 1
-		user << "You short out the product lock on [src]."
+		user << "You short out the product lock on [src] and unlock some items."

```

Comment:
It's just a message so people know what happens once you emag the machines, if you want me to remove it it's fine i guess